test/micro/org/openjdk/bench/vm/compiler/MergeStoreBench.java

```diff
@@ -1129,4 +1149,9 @@ public static void putShortLU(byte[] array, int offset, int c) {
         UNSAFE.putByte(array, address    , (byte) (c     ));
         UNSAFE.putByte(array, address + 1, (byte) (c >> 8));
     }
+
+    @Fork(value = 1, jvmArgsPrepend = {
```

Comment:
```suggestion
    @Fork(value = 1, jvmArgs = {
```
Can you make this change, and run the benchmarks again?
There was a recent JMH build script change, and all usages of `jvmArgsPrepend` in JMH tests were supposed to be changed to `jvmArgs`. I think in your case the flag is actually not applied.

Not sure if that is true, but it looks that way to me.

https://github.com/openjdk/jdk/pull/21800